floor/floor/controller/midi/manager.py

```diff
@@ -88,11 +88,15 @@ def on_midi_commands(self, peer, commands):
         for command in commands:
             func = mapping.get_function(command)
             if not func:
+                self.logger.info("Received unmapped command: {} {}".format(command[0], command[1]))
```

Comment:
I spent 30m trying to figure out why my AKAI wasn't working in dev mode before I realized I had not added a "--midi_mapping" flag to give the mapping.  At least this way we'll have some indication if something is trying to talk and we don't understand it.